src/routes.ts

```diff
@@ -6,16 +6,75 @@ import validateRequest from './middlewares/validationResults'
 const router = express.Router()
 
 router.route('/tasks')
+  /**
+   * POST /tasks
+   * @summary Creates a task.
+   * @bodyContent {CreateUpdateTask} application/json
+   * @bodyRequired
+   * @response 201 - Created
+   * @responseContent {Task} 201.application/json
+   * @response 400 - Validation error
+   */
   .post(validateRequest(CreateTaskDto, 'body'),
     task.createTask)
+  /**
+   * GET /tasks
+   * @summary Get all tasks
+   * @queryParam {integer} [skip=0] - The number of items to skip before starting to collect the result
+   * @queryParam {integer} [limit=20] - The numbers of items to return
+   * @queryParam {string} [status] - Filter items by status
+   * @queryParam {string} [importance] - Filter items by importance
+   * @response 200 - The list of the tasks
+   * @responseContent {TasksResponse} 200.application/json
+   */
   .get(validateRequest(GetAllTaskDto, 'query'),
     task.getAllTasks)
 
 router.route('/tasks/:id')
   .all(validateRequest(IdParam, 'params'))
+  /**
+   * GET /tasks/{taskId}
```

Comment:
It should be `id` instead of `taskId` because you are using `id` name in routes.